app/client/src/widgets/wds/WDSInputWidget/widget/index.tsx

```
@@ -1,5 +1,5 @@
 import React from "react";
-import { debounce, isNumber, merge, toString } from "lodash";
+import { isNumber, merge, toString } from "lodash";
```

Biome (1.9.4)

[error] Do not shadow the global "toString" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)